cmd/storage-rest-common.go

```diff
@@ -25,6 +25,7 @@ const (
 const (
 	storageRESTMethodDiskInfo             = "/diskinfo"
```

Comment:
since we added a new API we would have to update the internode storage API version, if we can avoid that i.e some how code reliably works without bloom filter then it should work. 

But if we update the storage API version we would break rolling upgrades, which is okay but I just want to make sure that is the intention here and its okay to do.

Reply:
There shouldn't really be any major problems if some parts aren't updated. 

An error will be logged and bloom filters are simply ignored until all parts of the cluster is upgraded.